aion/web/elm/src/Update.elm

```diff
@@ -27,7 +28,25 @@ update msg model =
             { model | user = response } ! []
 
         OnQuestionCreated response ->
-            model ! []
+            case response of
+                RemoteData.Success responseData ->
+                    let
+                        oldPanelData =
+                            model.panelData
+
+                        oldCreateQuestionForm =
+                            model.panelData.createQuestionForm
+
+                        newCreateQuestionForm =
+                            Forms.updateFormInput oldCreateQuestionForm "question" ""
+
+                        evenNewerCreateQuestionForm =
```

Comment:
Haven't checked that, but instead of these three createQuestionForm lets we could do with one using pipe chains. What about `createQuestionForm -> updateFormInput -> updateFormInput`?